src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregationManager.cs

```diff
@@ -11,6 +11,37 @@
 
 namespace System.Diagnostics.Metrics
 {
+    internal enum HistogramAggregationType
+    {
+        Default, // default exponential aggregation with quantiles
+        Base2Exponential, // Base 2 exponential histogram aggregation
+    }
+
+    /// <summary>
+    /// Configuration options for histogram aggregation.
+    /// </summary>
+    /// <remarks>
+    /// Currently this class support Bae 2 Exponential Histogram aggregation configuration. In the future this can be extended to support more aggregation types.
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#base2-exponential-bucket-histogram-aggregation

```suggestion
    /// Currently this class support Base2 Exponential Histogram aggregation configuration. In the future this can be extended to support more aggregation types.
```